openfisca_uk/calibration/losses/categories/budgetary_impact/of_income_tax_by_income_band.py

```diff
@@ -4,19 +4,22 @@
 from openfisca_uk.parameters import parameters
 
 
-class IncomeTaxRevenueByIncome(LossCategory):
-    weight = 1
+class BudgetaryImpactOfIncomeTaxByIncomeBand(LossCategory):
     label = "Income tax revenue by income"
-    parameter_folder = parameters.calibration.income_tax.revenue_by_income
+    parameter_folder = (
+        parameters.calibration.budgetary_impact.of_income_tax_by_income_band
+    )
 
     def get_loss_subcomponents(
         sim,
         household_weights,
         year,
     ):
-        brackets = IncomeTaxRevenueByIncome.parameter_folder.brackets[
-            :-3
-        ]  # Last-3 brackets are impossible due to FRS non-capture
+        brackets = (
+            BudgetaryImpactOfIncomeTaxByIncomeBand.parameter_folder.brackets[
+                :-3
+            ]
+        )  # Last-3 brackets are impossible due to FRS non-capture
```

Comment:
Do we need to drop them? Should produce the same result either way right?

Keeping them would allow for more directly demonstrating how future revisions to improve it (synthesis) improve loss, without having to make the loss function incomparable to previous openfisca-uk versions.

Reply:
True, changed.